.gitignore

```diff
@@ -0,0 +1,128 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
Start a discussion on which files we need to ignore on django and why we have to do let's see how we can benefit from it.

Reply:
Since the pychache files are already in a commit adding gitignore here is not enough, you have to clean the branch up so the files are ignored
Do the following:

`git rm -r --cached .`
`git add .`
`git commit -m "<commit_message_addressing_cleanup>"`

Reply:
Have created an issue on this. will have to asign it to someone so that it is resolved in the upcoming pull request.